deploy_localstack.py

```diff
@@ -0,0 +1,188 @@
+                Content={
+                    'S3Bucket': bucket_name,
+                    'S3Key': layer_key,
+                },
+                CompatibleRuntimes=['python3.9'],
+                Description=f"Layer with {layer_name} dependencies"
+            )
+            layer_arn = response['LayerVersionArn']
+            print(f"Lambda Layer created: {layer_arn}")
+            return layer_arn
+        except Exception as e:
+            print(f"Error creating Lambda Layer: {str(e)}")
+            raise
+
+    def create_lambda_function(self):
+        try:
+            bucket_name, key_name = self.upload_lambda_to_s3()
+
+            # Subir y publicar las capas
+            light_layer_arn = self.create_lambda_layer('lambda_layer_light')
+            heavy_layer_1_arn = self.create_lambda_layer('lambda_layer_heavy_part1')
+            heavy_layer_2_arn = self.create_lambda_layer('lambda_layer_heavy_part2')
+            heavy_layer_3_arn = self.create_lambda_layer('lambda_layer_heavy_part3')
+            heavy_layer_4_arn = self.create_lambda_layer('lambda_layer_heavy_part4')
+
+            try:
+                self.lambda_client.update_function_code(
+                    FunctionName='biotablero-search-api',
+                    S3Bucket=bucket_name,
+                    S3Key=key_name
+                )
+                print("Lambda function updated")
+            except ClientError as e:
+                if e.response['Error']['Code'] == 'ResourceNotFoundException':
+                    self.lambda_client.create_function(
+                        FunctionName='biotablero-search-api',
+                        Runtime='python3.9',
+                        Role='arn:aws:iam::000000000000:role/lambda-role',  # Role ficticio para LocalStack
+                        Handler='app.main.handler',
+                        Code={'S3Bucket': bucket_name, 'S3Key': key_name},
+                        Timeout=30,
+                        MemorySize=256,
+                        Environment={
+                            'Variables': {
+                                'STAGE': 'dev',
+                                'POSTGRES_HOST': 'postgres',
+                                'POSTGRES_PORT': '5432',
+                                'POSTGRES_USER': 'postgres',
+                                'POSTGRES_PASSWORD': 'postgres',
+                                'POSTGRES_DB': 'localdb'
+                            }
+                        },
+                        Layers=[light_layer_arn, heavy_layer_1_arn, heavy_layer_2_arn, heavy_layer_3_arn, heavy_layer_4_arn]
+                    )
+                    print("Lambda function created with Layers")
+                else:
+                    raise e
+        except Exception as e:
+            print(f"Error with Lambda function: {str(e)}")
+            raise
+
+    def create_api(self):
+        """Crear API Gateway"""
+        try:
+            # Crear API
```

Comment:
We generally avoid using comments to describe functions or parts of them. However, if you think it's necessary, please ensure they are written in only one language (English). This applies to several files in this PR.